Upgrade-Guide/op_version.md

```diff
@@ -20,3 +24,9 @@ For example, after upgrading to glusterfs-3.7.1, set op-version as:
 Note: 
 This is not mandatory, but advisable to have updated op-version if you want to make use of latest features in the updated gluster.
 
+###Client op-version
+
+To check op-version information for clients the following query can be used:
+
+    [root@~]#gluster volume status <all|VOLNAME> clients
```

Comment:
"When trying to set a volume option, it might happen that one or more of the connected clients cannot support the feature being set and might need to be upgraded to the op-version the cluster is currently running on.

To check op-version information for the connected clients and find the offending client, the following query can be used:

    [root@~]#gluster volume status <all|VOLNAME> clients

The respective clients can then be upgraded to the required version.

This information can also be used to make an informed decision while bumping up the op-version of a cluster, so that connected clients can still support all the new features provided by the upgraded cluster."

Is this better?